### PR TITLE
[action] replace TAOS-CI script for android build in API Level 34 @open sesame 04/04 20:31

### PR DESCRIPTION
**Pull request by niley7464**
This patch replaces TAOS-CI script for android build.

This PR requires https://github.com/nnstreamer/nnstreamer-android-resource/pull/41

### CODE REVIEW

File: java/android/nnstreamer/build.gradle [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion '28.0.3'
+    namespace "org.nnsuite.nnstreamer"
+    compileSdk 34
     defaultConfig {
         minSdkVersion 24
-        targetSdkVersion 28
+        targetSdkVersion 34
```
Comment (again4you) [inline java/android/nnstreamer/build.gradle:8]: Question)
The reason why `targetSdkVersion` is 34 is that it is minimal required version of Google Play Store. right?

File: .github/workflows/android.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        abi: ['armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64']
+    steps:
+    - uses: nttld/setup-ndk@v1
```
Comment (again4you) [inline .github/workflows/android.yml:16]: Question)
The latest version of setup-ndk is v1.4.2 and the version you uses is v1.0.0, which is released in 2021. Is there any reason to use v1?

**Review by again4you — COMMENTED** [review]
I just added some questions. Except that, it looks good to me.

File: .github/workflows/android.yml (hunk 2 above)
Comment (niley7464) [reply .github/workflows/android.yml:16]: https://github.com/nnstreamer/nnstreamer/blob/b251e9bc6ba87b66056d4bf2adc852038cb7e38e/.github/workflows/android.yml#L15

I used that version to use the same version as nnstreamer.

File: java/android/nnstreamer/build.gradle (hunk 1 above)
Comment (niley7464) [reply java/android/nnstreamer/build.gradle:8]: Yes we should use 34 version for Play Store.

**Review by wooksong — APPROVED** [review]
LG(reat)TM!

**Review by jaeyun-jung — APPROVED** [review]

**Comment by jaeyun-jung** [comment]
@gichan-jang please update taos-ci on aws to avoid android lib build error.

**Comment by gichan-jang** [comment]
I removed ml-api-android build from TAOS-CI.
But checklist is not properly updated yet.
Please do force push again. (If it's not refresh the TAOS-CI checklist, let's ignore it)

**Comment by jaeyun-jung** [comment]
Now we removed android build on taos, merge this PR.

**Pull request merged**